main.css

```diff
@@ -11,13 +12,15 @@ body{
     margin:0;
     padding: 0;  
```

Comment:
You can add box-sizing: border-box here.

Reply:
will add it